docker/wit.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:xenial
+
+RUN apt-get update && apt-get install -y \
+   openjdk-8-jdk \
+   software-properties-common \
+   git  
+
+RUN git clone --branch v0.10.1 https://github.com/sifive/wit.git
```

Comment:
Since this _is_ the `wit` repository, you should just do a copy from the current directory.

Reply:
Furthermore, you should use `make install` because it removes unneeded stuff like the tests. You can do `make install PREFIX=/path/to/install` to set the directory where it puts the code. Note that it installs to a directory with the version like `/path/to/install/0.10.1`. You can override the version with `version=whatever` if needed.